### ISSUE TITLE
v3  version missing the visualization of spaghettis in the rating curve results

### ISSUE THREAD

**Issue by IvanHeriver** [BaRatin-tools/BaRatinAGE#51]
Displaying rating curve spaghettis (samples) is no longer possible in v3 since it was judged not useful enough. However, it can be quite useful for educational purposes e.g. when teaching the BaRatin method. 

@benRenard  and @JeromeLeCoz, is it worth implementing?

**Comment by JeromeLeCoz** [comment]
I agree, interesting for understanding only. I'd say, it depends on the implementation cost. If reasonable, it seems worth it, if complicated,, we can drop it.

**Comment by benRenard** [comment]
I agree, it sits rather low in the priority list imho. However what would be more useful is the possibility to export spaghettis (total uncertainty) for the RC and the Q time series, which is not implemented yet I think, unless I miss it.
